### ISSUE TITLE
Running apt on java7 with EmpireAnnotationProcessorFactory raises warnings

### ISSUE THREAD

**Issue by Riduidel** [mhgrove/Empire#64]
When i run apt (java7 one) with empire factory, I got these error logs

     [WARNING] warning: The apt tool and its associated API are planned to be
     [WARNING] removed in the next major JDK release.  These features have been
     [WARNING] superseded by javac and the standardized annotation processing API,
     [WARNING] javax.annotation.processing and javax.lang.model.  Users are
     [WARNING] recommended to migrate to the annotation processing features of
     [WARNING] javac; see the javac man page for more information.

Seems like EmpireAnnotationProcessorFactory should consider an upgrade.

**Comment by mhgrove** [comment]
These packages are gone for Java 8.  Can probably do away with the annotation processors once #99 is complete

**Comment by nayangogoi744** [comment]
When I run this command (sudo apt -get install libjson-rpc-perl
) on my RHEL 6 machine I am getting the same error.I am using javac 1.7.0_45

**Comment by delocalizer** [comment]
You're in the wrong place for help with your issue. I assume you're trying to install a Perl module using the package manager command "apt-get" but have mis-spelled your intended command and got the message from the java Annotation Processing Tool "apt". Your deeper problem is that 'apt-get' is the Debian/Ubuntu package manager; the equivalent on RHEL 6 is called 'yum'. So you want something like "yum install <package>". I strongly recommend some training before proceeding. http://www.redhat.com/en/services/training/all-courses-exams

Date: Thu, 3 Sep 2015 03:15:34 -0700
